src/App.jsx

```diff
@@ -39,11 +39,28 @@ function App() {
     }
   }, [totalReadingTimeMs, window.localStorage.getItem("wpm")]);
 
+  const [messageList, setMessageList] = useState([]);
+  const [isDeleteMode, setIsDeleteMode] = useState(false);
+  const [prevArticleDatas, setPrevArticleDatas] = useState([]);
```

Comment:
```suggestion
  const [messageList, setMessageList] = useState([]);
  const [isDeleteMode, setIsDeleteMode] = useState(false);
  const [prevArticleDataList, setPrevArticleDataList] = useState([]);
```
messageList 는 메세지들을 담는 리스트라는 뜻이라고 이해됩니다!
일관성을 위해 아티클 데이터를 담는 `prevArticleDatas` 또한 `prevArticleDataList`로 변경하는거나 `messageList`를 `messages` 로 변경하는 것은 어떨까요?

Reply:
prevArticleDatas => prevArticleDataList
setPrevArticleDatas => setPrevArticleDataList
로 변경했습니다